### ISSUE TITLE
abp generate-proxy apiName in the generated TS service is always called 'Default'

### ISSUE THREAD

**Issue by git102347501** [abpframework/abp#13388]
### Documentation

#Env
ABP Version: 5.3.0
User Interface: Angular
DataProvider: Ef Core

First of all, I worked in env JSON is configured with the module API node, as follows:

`
apis: {
    default: {
      url: 'https://api.test.com',
      rootNamespace: 'my.test',
    },
    person: {
      url: 'https://api.person.test.com',
      rootNamespace: 'my.person',
    }
}
`

Then, in the angular project, I use the following command line to generate TS rest service request code:

`abp generate-proxy -m app -t ng --api-name person`

Finally, I got the generated code:

`
export class XXXService {
  apiName = 'Default';
  // ...
}

`
The question is, why is apiname always called default? This causes me to manually correct the apiname to person every time before I can use it normally. Is it because I use cli incorrectly?

**Comment by realLiangshiwei** [comment]
https://github.com/abpframework/abp/blob/e3e1779de6df5d26f01cdc8e99ac9cbcb3d24d3c/modules/identity/src/Volo.Abp.Identity.HttpApi/Volo/Abp/Identity/IdentityUserController.cs#L9

https://github.com/abpframework/abp/blob/e3e1779de6df5d26f01cdc8e99ac9cbcb3d24d3c/modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IdentityRemoteServiceConsts.cs#L5

**Comment by git102347501** [comment]
> https://github.com/abpframework/abp/blob/e3e1779de6df5d26f01cdc8e99ac9cbcb3d24d3c/modules/identity/src/Volo.Abp.Identity.HttpApi/Volo/Abp/Identity/IdentityUserController.cs#L9
> 
> https://github.com/abpframework/abp/blob/e3e1779de6df5d26f01cdc8e99ac9cbcb3d24d3c/modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IdentityRemoteServiceConsts.cs#L5

I used the above method. First, I made a feature identification in appservice, as follows:

`
[RemoteService(Name = "person")]
public class PersonAppService : BasicAppService, IPersonAppService
{
//..
}
`

Then start the project. I see that remoteservicename is still default
<img width="487" alt="WX20220724-183733@2x" src="https://user-images.githubusercontent.com/37917403/180643343-8e6965f4-d1b1-4fa3-9045-0271b9691273.png">

Is the change method not applicable to the **appservice layer**? For automatic controllers, identifying the controller means a lot of repetitive creation of meaningless controller work.

I tried to identify the **Area** feature on appservice, but I couldn't seem to find any implementation.
